### PR TITLE
[search] Readable name calculation is added into reverse geocoder tests.

### PR DESCRIPTION
**Pull request by milchakov**

### CODE REVIEW

**Comment by mpimenov** [comment]
s/readable/Readable/

The correct tag is [search].

File: map/map_tests/address_tests.cpp [hunk 1]
```diff
@@ -4,27 +4,47 @@
...
-void TestAddress(ReverseGeocoder & coder, ms::LatLon const & ll, string const & street,
-                 string const & houseNumber)
+void TestAddress(ReverseGeocoder & coder, ms::LatLon const & ll, std::string const & street,
+                 std::string const & houseNumber)
 {
   ReverseGeocoder::Address addr;
   coder.GetNearbyAddress(MercatorBounds::FromLatLon(ll), addr);
 
-  string const key =
+  std::string const etalonKey =
```
Comment (mpimenov) [inline map/map_tests/address_tests.cpp]: s/etalon/expected/
or
s/etalon/want/

File: map/map_tests/bookmarks_test.cpp [hunk 2]
```diff
@@ -13,13 +15,16 @@
 #include "platform/platform.hpp"
 #include "platform/preferred_languages.hpp"
 
+#include "coding/string_utf8_multilang.hpp"
+
```
Comment (mpimenov) [inline map/map_tests/bookmarks_test.cpp]: лишняя пустая

File: map/map_tests/bookmarks_test.cpp [hunk 3]
```diff
@@ -438,12 +443,17 @@ UNIT_TEST(Bookmarks_Getting)
 
 namespace
 {
-void CheckPlace(Framework const & fm, double lat, double lon, string const & street,
-                string const & houseNumber)
+void CheckPlace(Framework const & fm, std::shared_ptr<MwmInfo> mwmInfo, double lat, double lon,
+                StringUtf8Multilang const & streetNames, string const & houseNumber)
```
Comment (mpimenov) [inline map/map_tests/bookmarks_test.cpp]: std::string

Comment (mpimenov) [reply map/map_tests/bookmarks_test.cpp]: or, even better, drop the std:: from shared_ptr

File: map/map_tests/bookmarks_test.cpp [hunk 4]
```diff
@@ -438,12 +443,17 @@ UNIT_TEST(Bookmarks_Getting)
...
 {
   auto const info = fm.GetAddressAtPoint(MercatorBounds::FromLatLon(lat, lon));
 
-  TEST_EQUAL(info.GetStreetName(), street, ());
+  std::string streetName;
```
Comment (mpimenov) [inline map/map_tests/bookmarks_test.cpp]: this file uses namespace std

File: map/map_tests/address_tests.cpp (hunk 1 above)
Comment (milchakov) [reply map/map_tests/address_tests.cpp]: заменил на expected

File: map/map_tests/bookmarks_test.cpp (hunk 2 above)
Comment (milchakov) [reply map/map_tests/bookmarks_test.cpp]: перенес инклуд

File: map/map_tests/bookmarks_test.cpp (hunk 3 above)
Comment (milchakov) [reply map/map_tests/bookmarks_test.cpp]: готово

File: map/map_tests/bookmarks_test.cpp (hunk 4 above)
Comment (milchakov) [reply map/map_tests/bookmarks_test.cpp]: удалил

**Comment by milchakov** [comment]
@mpimenov PTAL

**Pull request merged**